<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmOperCuentas</class>
 <widget class="QDialog" name="frmOperCuentas">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nuevo movimiento de cuenta</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/xulpmoney/document-edit.png</normaloff>:/xulpmoney/document-edit.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lblTitulo">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 128, 0);</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Selecciona la fecha del movimiento</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCalendarWidget" name="calendar">
         <property name="firstDayOfWeek">
          <enum>Qt::Monday</enum>
         </property>
         <property name="verticalHeaderFormat">
          <enum>QCalendarWidget::NoVerticalHeader</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Selecciona una cuenta bancaria</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbCuentas"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Selecciona un concepto</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbConceptos"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Introduce un importe</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="myQLineEdit" name="txtImporte">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Introduce un comentario</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtComentario">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="cmd">
       <property name="text">
        <string>Guardar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>myQLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">myqlineedit</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images/xulpymoney.qrc"/>
 </resources>
 <connections/>
</ui>
